### ISSUE TITLE
Failed to read module directory

### ISSUE THREAD

**Issue by konstantin-recurly** [antonbabenko/pre-commit-terraform#84]
I had to upgrade terraform-docs because of issue with a null value in variables.tf 
https://github.com/segmentio/terraform-docs/issues/161
Now pre-commit complains about the directory.

Here is some information about the versions.
pre-commit-config.yaml
```Bash
- repo: git://github.com/antonbabenko/pre-commit-terraform
  rev: v1.22.0
  hooks:
    - id: terraform_fmt
    - id: terraform_docs
```
run pre-commit run -a
```Bash
pre-commit run -a
Terraform fmt............................................................Passed
Terraform docs...........................................................Failed
hookid: terraform_docs

2020/01/14 16:27:41 Failed to read module directory: Module directory /var/folders/_v/n250j8397bj273wj64d3rz180000gn/T//terraform-docs-TQYHbozJA7.tf does not exist or cannot be read.
```
terraform-docs version
```Bash
terraform-docs version
terraform-docs version v0.8.0-rc.2 d52122d darwin/amd64 BuildDate: 2020-01-12T20:38:24+0000
```

**Comment by maxgio92** [comment]
I'm experiencing the same issue:

```
Terraform validate without variables.....................................Passed
Terraform fmt............................................................Passed
Terraform docs...........................................................Failed
- hook id: terraform_docs
- exit code: 1

2020/01/15 18:22:39 Failed to read module directory: Module directory /tmp/terraform-docs-oXFApyympv.tf does not exist or cannot be read.
```

**Comment by lifeofguenter** [comment]
terraform-docs 0.8.0 has been released on `brew`, so unfortunately anyone installing/upgrading will not be able to use this pre-commit tool as of current.

**Comment by antonbabenko** [comment]
Resolved in #85 

https://github.com/antonbabenko/pre-commit-terraform/pull/85#issuecomment-576616846
